core/capabilities/ccip/ccipsolana/extradatadecoder.go

```diff
@@ -26,6 +29,48 @@ var (
 	evmExtraArgsV2Tag = hexutil.MustDecode("0x181dcf10")
 )
 
+type ExtraDataCodec struct{}
+
+func NewExtraDataCodec() ExtraDataCodec {
+	return ExtraDataCodec{}
+}
+
+func (c ExtraDataCodec) DecodeExtraArgs(extraArgs cciptypes.Bytes, sourceChainSelector cciptypes.ChainSelector) (map[string]any, error) {
+	if len(extraArgs) == 0 {
+		// return empty map if extraArgs is empty
+		return nil, nil
+	}
```

Comment:
I'm not sure extraArgs is allowed to be empty on SVM based on a recent discussion \cc @RensR ?